vcpkg.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "freerdp",
+    "version-string": "3.3.0",
+    "description": "A free implementation of the Remote Desktop Protocol (RDP)",
+    "homepage": "https://github.com/FreeRDP/FreeRDP",
+    "license": "Apache-2.0",
+
+    "dependencies": [
+        "zlib",
+		"openssl"
+    ],
+
+    "default-features": [
+    ],
+
+    "features": {
+		"all": {
+			"description": "Enable all features",
+            "dependencies": [
+				"openssl",
+				"openh264",
+				"opus",
+				"ffmpeg",
+				"cjson",
+				"libusb",
+				"opencl",
+				"cairo",
+				"uriparser",
+				"sdl2",
+				"sdl2-ttf"
+			]
+		},
+		"freerdp-with-ffmpeg": {
+			"description": "Enable the feature of freerdp library with ffmpeg",
+            "dependencies": [
+			    "openh264",
```

Comment:
`openh264` and `ffmpeg` are alternatives.
while it is possible to compile against both, it is not recommended.

Reply:
Which one is officially recommended？

Reply:
depends on what licensing terms you can be comfortable with.
* `x264` has better quality but is `GPL`
* `openh264` is `BSD`

Reply:
I don't care about licensing agreements. I'm using openh264 for the time being.